packages/suite-web/e2e/tests/backup/t2-fail.test.ts

```diff
@@ -39,3 +39,5 @@ describe('Backup fail', () => {
         );
     });
 });
+
+export {};
```

Comment:
It's necessary to add this to end every test because we type-checking these test by tsconfig.json that is used also for standart source code in `src/`. We need little bit different config for tests but only way how to make it work correct is to create package just for e2e tests. Which is good idea anyway.